espree.js

```diff
@@ -4398,7 +4403,7 @@ function parseFunctionSourceElements() {
     oldInIteration = state.inIteration;
     oldInSwitch = state.inSwitch;
     oldInFunctionBody = state.inFunctionBody;
-    oldParenthesisCount = state.parenthesizedCount;
```

Comment:
This seemed like a typo, because half the time it's spelled differently, so I changed it. `parenthesizedCount` vs `parenthesisCount`. They are the same thing right?